index.html

```diff
@@ -5,19 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <script src="./navigation.js" defer></script>
     <title>Awesome Books</title>
 </head>
 <body>
-    <h1 id="list-title">Awesome Books</h1> 
-    <div id="book-list">
-    <!--Booklist added in javascript  -->
-    </div>
-    <form id="book-form">
-        <h2 id="form-title">Add new book</h2>
-        <input type="text" id="title" placeholder="Title" required>
-        <input type="text" id="author" placeholder="Author" required>
-        <button type="submit">Add</button>
-    </form>
+    <header>
+        <nav>Awesome book
+            <ul id="navbar">
+                <li class="nav-btn" data-id="book-list-section">Book-list</li> 
+                <li class="nav-btn" data-id="form">Add new</a></li> 
+                <li class="nav-btn" data-id="Contact">Contact</li> 
+            </ul>
+        </nav>
```

Comment:
**OPTIONAL** - You have done well so far, but there are differences between your project and the wireframe, your website is missing the clock/time that is visible just below the navbar. Please fix this 🙏 

Your project:
![everv](https://user-images.githubusercontent.com/73607512/227139030-04b6708b-b04e-4ac1-9584-8e5daea5e073.PNG)

Reply:
Optional issue fixed in recent commits.